src/MIDebugEngine/Engine.Impl/BreakpointManager.cs

```diff
@@ -221,13 +220,23 @@ public AD7BoundBreakpoint[] FindHitBreakpoints(string bkptno, ulong addr, /*OPTI
                     continue;
                 }
 
-                hitBoundBreakpoints.Add(currBoundBp);
-            }
+                string currBkptNo = currBoundBp.PendingBreakpoint.BreakpointId;
+                TupleValue bkpt = await _engine.DebuggedProcess.MICommandFactory.BreakInfo(currBkptNo);
+                uint? hitTimes = bkpt.TryFindUint("times");
```

Comment:
Does this tell us if the breakpoint has ever been hit? Or does 'times' get reset every time we continue?